Week2/project/index.js

```diff
@@ -4,7 +4,106 @@
 
   If the timer is running, the user can 't change the session length anymore
   Use at least 3 functions
-  Display minutes and seconds
-  If the timer finishes the timer should be replaced by the message: Time 's up!
+  disabled minutes and seconds
+  If the timer finishes the timer should be replaced by the message: Time 's up1!
  * 
- */
\ No newline at end of file
+ */
+
+'use strict';
+
+//getting all the needed elements:
+const upBtn = document.getElementById('add-time');
+const dwon = document.getElementById('decrease-time');
+const setSessionLength = document.getElementById('duration-select');
+const minutesIndicator = document.getElementById('minutes');
+const secondsIndicator = document.getElementById('seconds');
+const startBtn = document.getElementById('start');
+const pauseBtn = document.getElementById('pause');
+const restartBtn = document.getElementById('stop');
+let countDownInt;
+const interval = 1000;
+const time = new Date();
+let minute = 25;
+
+//the follwing function for adding minutes and remove 
+function add(){
+  minute++;
+  setSessionLength.textContent=(minute);
+  reset()
+}
+function remove(){
+  if (minute > 1) {
+    minute--;
+    setSessionLength.textContent = minute;
+    
+  }
+  reset()
+}
+//this function for set minutes to new minutes
+ function reset(){
+  time.setMinutes(minute);
+  time.setSeconds(0);
+  showTime();
+ }
+ //this function to display the new time 
+  function showTime(){
+    minutesIndicator.textContent=disabled(time.getMinutes());
+    secondsIndicator.textContent=disabled(time.getSeconds());
+    
+  }
+  //thid function to display the time as two digits format
+  function disabled (number){
```

Comment:
why is this called disabled?

Reply:
ooh i meant to call it display to show the number but i made a mistake with its name 
```suggestion
  function display (number){
```